chapter2 presentation

shmuhammadd opened the pull request.

shmuhammadd committed:
doc: introduction session

shmuhammadd committed:
doc: chapter2

jonthegeek committed:
Merge branch 'main' into chapter2

jonthegeek reviewed (changes requested):
Add `lexicon` to the DESCRIPTION file. You can do so quickly by typing this in your console:
`usethis::use_package("lexicon")`

Add names to all of the R blocks, in the form 02-blockname (such as 02-setup). That way, when there's an error, it's much easier to locate.

jonthegeek committed:
Merge branch 'main' into chapter2

jonthegeek committed:
Fix sentiment evaluation.

Unfortunately we can't run the sentiment stuff in the bookdown due to licensing, so we need to turn eval off for a bunch of things.

jonthegeek reviewed (approved):
Looks good! I had to turn off evaluation for some things (the sentiment stuff requires a human acknowledgement for licensing reasons, so it can't run online), but otherwise everything looks good!
